basics/answers.md

```diff
@@ -92,7 +92,18 @@ isn't exactly like mine but it works just fine it's ok, and you can
         print("Access denied.")
     ```
 
-    Again, this is not a good way to ask a real password from the user.
+    Again, this is not a good way to ask a real password from the user. 
+
+7. Here, we create  a new variable `pallindrome_check`.We use this variable to store the  reversed  string(of`pallindrome_input`). <br>Then we check if the string that was given(`pallindrome_input`)
+   and the reversed sting(`pallindrome_check`) is the same using a simple `if loop`
+   ```python
+   pallindrome_input=input("Type the number to check:") #to get the input from user
+   pallindrome_check=pallindrome_input[::-1] #Reverses the string
+   if pallindrome_input==pallindrome_check:
+    print(f"This number is a pallindrome")
```

Comment:
Please indent these with 4 spaces.

Your markdown list item (starting with `7.`) is indented with 3 spaces. That too should be indented with 4 spaces, so you get 4 spaces for most lines and 4+4 lines for indented code.